README.md

```diff
@@ -8,17 +8,6 @@
 
 ![Ui](docs/images/Ui.png)
 
-* This project is based on the `AddressBook-Level3 (AB3)` project created by the [SE-EDU initiative](https://se-education.org).
-* This project, `ContactsForGood (CFG)` is an extension of the `AB3` project targeted towards NGO administrators.'
-* It contains improvements and new features which are of specific benefit to NGO administrators and their work.
-* The details of the original AB3 project are included below:
-  * This is **a sample project for Software Engineering (SE) students**.<br>
-    Example usages:
-    * as a starting point of a course project (as opposed to writing everything from scratch)
-    * as a case study
-  * The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-    * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-    * It comes with a **reasonable level of user and developer documentation**.
-  * It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-  * For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-  * This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org/#contributing-to-se-edu) for more info.
+* This project, `ContactsForGood (CFG)` is a CLI-based address book targeted towards NGO administrators.
```

Comment:
Would calling it a CLI-based contact management application designed specifically for NGO administrators be better?

Reply:
Sounds good, will push the new change soon